### PR TITLE
#824 - Background Colors

### PR DESCRIPTION
**Pull request by kevincrafts**

### CODE REVIEW

**Comment by alexfinnarn** [comment]
I can see the examples but not the options in the select list. Cleared the cache and tried to test on an existing hero unit. 
screen shot 2017-03-23 at 11 58 14 am

File: modules/custom/cu_core/cu_core.install [hunk 1]
```diff
@@ -679,6 +679,15 @@ function cu_core_update_7019() {
 }
 
 /**
+ * Core Update 7020 for Express 2.4.0.
+ *
+ * Revert cu_block_theme feature.
+ */
+function cu_core_update_7020() {
+  cu_core_revert_features(array('cu_block_theme'));
```
Comment (alexfinnarn) [inline modules/custom/cu_core/cu_core.install]: should be `cu_core_revert_features(array('cu_block_theme' => 'cu_block_theme'));` to work

**Comment by cathysnider** [comment]
Text color problem: A Light Gray BOX or White BOX placed on a Dark Gray BLOCK has white text. I.E. it is unreadable. 
boxcolorsonblocks1

**Comment by cathysnider** [comment]
https://www-dev.colorado.edu/feature/824

**Pull request merged**